lib/queries.js

```diff
@@ -171,16 +242,297 @@ export async function getAllOvoAndKboCouples() {
+                <http://xmlns.com/foaf/0.1/page> ?website .
+      
+      FILTER(?contact = ${sparqlEscapeUri(identifiers.contactPointUri)})
+    }
+  } 
+  `;
+  await update(updateStr);
+
+  //update kboUnit
+  updateStr = `
+  DELETE{
+    GRAPH ?g{
+      ?kboUnit a <http://mu.semte.ch/vocabularies/ext/KboEenheid> ;
+                <http://mu.semte.ch/vocabularies/ext/rechtsvorm> ?rechtsVorm ;
+                <http://mu.semte.ch/vocabularies/ext/shortName> ?shortName ;
+                <http://mu.semte.ch/vocabularies/ext/startDate> ?startDate ;
+                <http://www.w3.org/2004/02/skos/core#prefLabel> ?orgName ;
+                <http://www.w3.org/2004/02/skos/core#altLabel>  ?commercialName ;
+                <https://data.lblod.info/vocabularies/generiek/geplandeEindDatum> ?changeTime ;
+                <http://www.w3.org/ns/regorg#orgStatus> ?activeState .
+    }                                
+ }INSERT{
+  graph ?g{ 
+    ?kboUnit a <http://mu.semte.ch/vocabularies/ext/KboEenheid> ;
+              <http://mu.semte.ch/vocabularies/ext/rechtsvorm> ${sparqlEscapeString(
+                newKboInfo.rechtsvorm
+              )} ;
+              <http://mu.semte.ch/vocabularies/ext/shortName> ${sparqlEscapeString(
+                newKboInfo.shortName
+              )} ;
+              <http://mu.semte.ch/vocabularies/ext/startDate> ${sparqlEscapeString(
+                newKboInfo.startDate
+              )} ;
+              <http://www.w3.org/2004/02/skos/core#prefLabel> ${sparqlEscapeString(
+                newKboInfo.organisationName
+              )} ;
+              <http://www.w3.org/2004/02/skos/core#altLabel> ${sparqlEscapeString(
+                newKboInfo.formalName
+              )} ;
+              <https://data.lblod.info/vocabularies/generiek/geplandeEindDatum> ${sparqlEscapeString(
+                newKboInfo.changeTime
+              )} ;
+              <http://www.w3.org/ns/regorg#orgStatus> ${sparqlEscapeUri(
+                newKboInfo.activeState
+              )} .
+  }
+}
+WHERE{
+  VALUES ?g {
+    <http://mu.semte.ch/graphs/administrative-unit> 
+  }
+  GRAPH ?g {
+    ?kboUnit a <http://mu.semte.ch/vocabularies/ext/KboEenheid> ;
+                <http://mu.semte.ch/vocabularies/ext/rechtsvorm> ?rechtsVorm ;
+                <http://mu.semte.ch/vocabularies/ext/shortName> ?shortName ;
+                <http://mu.semte.ch/vocabularies/ext/startDate> ?startDate ;
+                <http://www.w3.org/2004/02/skos/core#prefLabel> ?orgName ;
+                <http://www.w3.org/2004/02/skos/core#altLabel>  ?commercialName ;
+                <https://data.lblod.info/vocabularies/generiek/geplandeEindDatum> ?changeTime ;
+                <http://www.w3.org/ns/regorg#orgStatus> ?activeState .
+    
+    FILTER(?kboUnit = ${sparqlEscapeUri(identifiers.kboUnit)})
+  }
+} 
+  `;
+  await update(updateStr);
+}
+
+export async function linkAbbOrgToKboOrg(abbOrgUri, kboOrgUri) {
+  let updateStr = `
```

Comment:
**Issue:** In order to avoid wrong insert, I would add a guard to check if abbOrgUri and kboOrgUri have value.